### PR TITLE
Mention ERC-831 ENS condition to prevent bad implementations

### PR DESCRIPTION
**Pull request by ligi**
as requested by @3esmit
context:
https://gitter.im/ethereum/EIPs?at=5daf62587477946badafe941
https://github.com/status-im/status-react/issues/9238

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 681 requires approval from one of (daniel@ethereum.org)

File: EIPS/eip-681.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ For the syntax of ENS_NAME, please consult ERC #137 defining Ethereum Name Servi
 
 ### Semantics
 
-`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact.
+`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact. Make sure to respect ERC #831 and add the "pay-" prefix in case you use ENS here.
```
Comment (axic) [inline EIPS/eip-681.md]: @ligi can you please change "ERC #831" to "ERC-831" and make it a link to https://eips.ethereum.org/EIPS/eip-831

Comment (axic) [reply EIPS/eip-681.md]: Oh I see you have followed the style of the draft, however this is the recommendation as per EIP-1.

**Comment by axic** [comment]
@nagydani can you please review?

**Comment by axic** [comment]
Ping @nagydani

**Comment by axic** [comment]
@ligi @nagydani any progress on this one?

**Pull request closed without merging**